torch/nn/modules/linear.py

```diff
@@ -115,4 +116,99 @@ def __repr__(self):
             + ', in2_features=' + str(self.in2_features) \
             + ', out_features=' + str(self.out_features) + ')'
 
+
+class NoisyLinear(Module):
+    """Applies a noisy linear transformation to the incoming data:
+        :math:`y = (mu_w + sigma_w \cdot epsilon_w)x
+            + mu_b + sigma_b \cdot epsilon_b`
```

Comment:
Add a note in the docs to note the equation used during evaluation (just the mu's).